lib/teracy-dev-essential/config/host_manager.rb

```diff
@@ -31,9 +35,48 @@ def configure_node(settings, config)
 
       private
 
+      def configure_ip_display(config, settings)
+        extension_lookup_path = TeracyDev::Util.extension_lookup_path(settings, 'teracy-dev-essential')
+
+        networks = get_all_configured_networks(settings)
+
+        # default VM network (127.0.0.1) is at 0 index
+        # we use the latest defined network IP
+        host_ip_index = networks.length
+
+        # if there are no networks defined
+        # init a default private network with dynamic ip
+        if host_ip_index == 0
+          config.vm.network "private_network", type: "dhcp"
+
+          host_ip_index = 1
+        end
+
+        config.vm.provision "shell",
+          run: "always",
+          args: [host_ip_index],
+          path: "#{extension_lookup_path}/teracy-dev-essential/provisioners/shell/ip_display.sh",
+          name: "Display IP"
+
+        host_ip_index
+      end
+
+      def get_all_configured_networks(settings)
+        networks = []
+
+        settings['nodes'].each do |node|
+          networks = node['vm']['networks'].each do |network|
+            networks << network
+          end if node['vm']['networks']
+        end
+
+        @logger.debug("networks #{pp(networks)}")
```

Comment:
remove pp here because it will always display this debug message even the LOG_LEVEL is above debug